WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -400,14 +398,25 @@ private extension NotificationsViewController {
         tableViewHandler = handler
     }
 
-    func setupRatingsView() {
-        precondition(ratingsView != nil)
+    func setupInlinePrompt() {
+        precondition(inlinePromptView != nil)
 
-        ratingsView.delegate = self
-        ratingsView.alpha = WPAlphaZero
+        inlinePromptView.alpha = WPAlphaZero
 
         // this allows the selector to move to the top
-        ratingsSpaceConstraint.isActive = false
+        inlinePromptSpaceConstraint.isActive = false
+
+        if Feature.enabled(.primeForPush) && !userDefaults.notificationPrimerInlineWasAcknowledged {
```

Comment:
A nice-to-have-but-not-dealbraker: It'd be nice to hide the implementation details of _when_ the push prime should be displayed — perhaps an extension on `PushNotificationManager`?